### ISSUE TITLE
请问使用whisper转录时，安装了cudnn-8.9，已有libcudnn_ops.so.8还要求寻找libcudnn_ops.so.9.1.0，需要重装cudnn吗？

### ISSUE THREAD

**Issue by ann578** [Kedreamix/Linly-Dubbing#41]
ReproducibilityWarning: TensorFloat-32 (TF32) has been disabled as it might lead to reproducibility issues and lower accuracy.
It can be re-enabled by calling
   >>> import torch
   >>> torch.backends.cuda.matmul.allow_tf32 = True
   >>> torch.backends.cudnn.allow_tf32 = True
See https://github.com/pyannote/pyannote-audio/issues/1370 for more details.

  warnings.warn(
Unable to load any of {libcudnn_ops.so.9.1.0, libcudnn_ops.so.9.1, libcudnn_ops.so.9, libcudnn_ops.so}
Invalid handle. Cannot load symbol cudnnCreateTensorDescriptor

**Comment by MoaMoa90** [comment]
same problem :(
